draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1115,37 +1162,32 @@ The function map2curve\_icart(alpha) implements the Icart encoding method from {
 Preconditions: An elliptic curve over F, such that p>3 and q=p^m=2 (mod 3), or
 p=2 (mod 3) and odd m.
 
-Input: alpha, an octet string to be hashed.
-
 Constants: A and B, the parameters of the Weierstrass curve.
 
-Output: (x, y), a point on E.
-
 Sign of y: this encoding does not compute a square root, so there
 is no ambiguity regarding the sign of y.
 
-Exceptions: The only exceptional case is hash2base(alpha) == 0.
-Implementations must detect this case by testing whether u = 0
+Exceptions: The only exceptional case is u == 0.
```

Comment:
>  The only exceptional case is u == 0.

 The only exceptional case is u = 0.

Reply:
This is a pervasive style question in the document. Pushed to #105.